src/ext/dispatch.c

```diff
@@ -378,7 +387,15 @@ int ddtrace_wrap_fcall(zend_execute_data *execute_data TSRMLS_DC) {
 #endif
 
     wrap_and_run(execute_data, dispatch TSRMLS_CC);
-
+#if PHP_VERSION_ID >= 70000
+    if (DDTRACE_G(original_context).this) {
+#if PHP_VERSION_ID < 70300
```

Comment:
Maybe we should backport GC_DELREF to earlier PHP versions?

Reply:
I've seen both approaches in 7.3, but yeah, backporting GC_DELREF macro seems like a good way to make the code a bit cleaner.

Reply:
The GC_ADDREF macro would get used in at least a few places too. Let's leave that for another PR.